libs/vkd3d/device.c

```diff
@@ -2174,6 +2184,15 @@ static HRESULT vkd3d_init_device_caps(struct d3d12_device *device,
     physical_device_info->vulkan_1_1_features.protectedMemory = VK_FALSE;
     physical_device_info->vulkan_1_1_features.samplerYcbcrConversion = VK_FALSE;
 
+    physical_device_info->vulkan_1_2_features.shaderInputAttachmentArrayDynamicIndexing = VK_FALSE;
+    physical_device_info->vulkan_1_2_features.shaderInputAttachmentArrayNonUniformIndexing = VK_FALSE;
+    physical_device_info->vulkan_1_2_features.bufferDeviceAddressCaptureReplay = VK_FALSE;
+    physical_device_info->vulkan_1_2_features.bufferDeviceAddressMultiDevice = VK_FALSE;
+    physical_device_info->vulkan_1_2_features.vulkanMemoryModelAvailabilityVisibilityChains = VK_FALSE;
```

Comment:
We're not using vulkan memory model either yet. There's a non-zero risk it can affect some driver's compiler.

8-bit storage is also unused.

Reply:
There are misc scenarios where unrelated features affect compilers globally (e.g. on NV there was a bug where enabling mesh shader extension affected PSO keys for some reason), or fragment shading rate affected all PSOs in RADV even if the PSO didn't use it.

I think the main argument is consistency though, since we're disabling features we're not using here to begin with, might as well do it consistently.